.github/CHANGELOG.md

```diff
@@ -11,9 +11,11 @@
 
 * New function to check if a matrix is symplectic `is_symplectic`. [#334](https://github.com/XanaduAI/thewalrus/pull/334).
 
+* Adds support for Python 3.10. [#337](https://github.com/XanaduAI/thewalrus/pull/337)
```

Comment:
Would you consider this a new feature? Or is it just an improvement (e.g., a package description update)?

Reply:
Good point! As long as we can argue it is a "new feature", I don't see a problem with including it in that section.